test/jest/kitFilterHelper.spec.ts

```diff
@@ -386,4 +386,92 @@ describe('FilterHashingUtilities', () => {
             expect(resultHash).toBe(expectedHash);
         });
     });
-});
+
+    describe('#filterUserAttributes', () => {
+        it('should return the original user attributes if no filter list is provided', () => {
+            const userAttributes = {
+                'foo-key': 'foo-value',
+                'bar-key': 'bar-value',
+                'baz-key': 'baz-value',
+            };
+
+            const result = KitFilterHelper.filterUserAttributes(userAttributes, []);
+
+            expect(result).toEqual(userAttributes);
+        });
+
+        it('should filter user attributes', () => {
+            const userAttributes = {
+                'foo-key': 'foo-value',
+                'bar-key': 'bar-value',
+                'baz-key': 'baz-value',
+            };
+
+            const fooFilter = KitFilterHelper.hashUserAttribute('foo-key');
+            const barFilter = KitFilterHelper.hashUserAttribute('bar-key');
+
+            const result = KitFilterHelper.filterUserAttributes(userAttributes, [
+                fooFilter,
+                barFilter,
+            ]);
+
+            expect(result).toEqual({
+                'baz-key': 'baz-value',
+            });
+        });
+    });
+
+    describe('#filterUserIdentities', () => {
+        it('should return the original user identities if no filter list is provided', () => {
+            const userIdentities = {
+                'foo-key': 'foo-value',
+                'bar-key': 'bar-value',
+                'baz-key': 'baz-value',
```

Comment:
these shoudl be userIdentities
```suggestion
                'other': 'foo-value',
                'customerid': 'bar-value',
                'facebook': 'baz-value',
```